build.gradle.kts

```diff
@@ -247,6 +251,50 @@ fun Project.setupPublishing() {
                 artifact(sourcesJar)
 
                 pom {
+                    name.set("FirebaseUI ${project.name.capitalize()}")
+                    description.set("Firebase UI for Android")
+                    url.set("https://github.com/firebase/FirebaseUI-Android")
+
+                    organization {
+                        name.set("Firebase")
+                        url.set("https://github.com/firebase")
+                    }
+
+                    scm {
+                        val scmUrl = "scm:git:git@github.com/firebase/firebaseui-android.git"
+                        connection.set(scmUrl)
+                        developerConnection.set(scmUrl)
+                        url.set(this@pom.url)
+                        tag.set("HEAD")
+                    }
+
+                    developers {
+                        developer {
+                            id.set("samtstern")
+                            name.set("Sam Stern")
+                            email.set("samstern@google.com")
+                            organization.set("Firebase")
+                            organizationUrl.set("https://firebase.google.com")
+                            roles.set(listOf("Project-Administrator", "Developer"))
+                            timezone.set("-8")
+                        }
```

Comment:
@lsirac If you can send me a developer DSL like this, I'll add you to the list too. 👍

Reply:
BTW @lsirac totally optional, if you don't want to expose your email address or name you can skip.